content/hi/agile_software_development.md

```diff
@@ -0,0 +1,14 @@
+---
+title: एजाइल सॉफ्टवेयर विकास
+status: Completed
+category: अवधारणा
+---
+
+## यह क्या है
+प्रथाओं का एक सेट जो पुनरावृत्त विकास चक्रों और स्वयं-संगठित टीमों पर जोर देता है। जलप्रपात जैसी परियोजनाओं के विपरीत जहां मूल्य केवल एक परियोजना के अंत में उत्पन्न होता है, एजाइल सॉफ्टवेयर विकास मूल्य की निरंतर, वृद्धिशील वितरण और प्रक्रिया के विकासवादी सुधार पर केंद्रित है।
+
+## समस्या
+एक सॉफ्टवेयर प्रोजेक्ट में सभी हितधारकों के लिए आवश्यकताओं को परिभाषित करना, संप्रेषित करना और समझना बहुत कठिन है। फिर भी, ग्राहक चाहते हैं कि उनके सॉफ्टवेयर प्रोजेक्ट समय पर, अच्छी गुणवत्ता में, बजट पर और दायरे में वितरित किए जाएं। अपनी चक्रीय प्रकृति के साथ, एजाइल सॉफ्टवेयर विकास आवश्यकताओं के निरंतर अनुकूलन और अन्य सभी परिस्थितियों के लिए तेजी से अनुकूलन को सक्षम बनाता है, जो जलप्रपात जैसी रणनीतियों के विपरीत है।
+
+## समाधान
+फुर्तीली सॉफ्टवेयर विकास में पारंपरिक रणनीतियों के सभी चरण, जैसे आवश्यकताएं इंजीनियरिंग, योजना, कार्यान्वयन, समीक्षा, परीक्षण और वितरण शामिल हैं। सबसे बड़ा अंतर यह है कि एक सॉफ्टवेयर प्रोजेक्ट के पूरे समय को पुनरावृत्तियों में काट दिया जाता है, जिनमें से प्रत्येक में वे सभी चरण होते हैं। प्रत्येक पुनरावृत्ति के बाद, ग्राहक के साथ बनाए गए मूल्य की समीक्षा की जा सकती है और आवश्यकताओं को अंतिम लक्ष्य की ओर समायोजित किया जा सकता है। इसके अतिरिक्त विकास दल(development team)पूर्व-निरीक्षण करता है कि प्रक्रिया को बेहतर बनाने के लिए किन कार्यों को करना है।
```

Comment:
Instead of `फुर्तीली`, `एजाइल ` would be better.

Reply:
Ah! must have missed it. Thankyou for pointing out :)